package.json

```
@@ -15,7 +15,7 @@
     "codecov": "^3.1.0",
     "mocha": "^5.2.0",
     "nyc": "^13.1.0",
-    "request": "^2.46.0"
+    "request": "2.46.0"
```

```suggestion
    "request": "^2.46.0"
```

I really think you did that by accident. Agree?

👍